### PR TITLE
LTP: Log kernel-rt package

### PR DESCRIPTION
**Pull request by pevik**
Verification run:
- http://quasar.suse.cz/tests/1631#step/boot_ltp/71
- http://quasar.suse.cz/tests/1632#step/boot_ltp/74

### CODE REVIEW

File: lib/LTP/utils.pm [hunk 1]
```diff
@@ -121,6 +122,8 @@ sub log_versions {
     record_info('KERNEL DEFAULT PKG', script_output("cat $kernel_pkg_log", proceed_on_failure => 1));
     record_info('KERNEL EXTRA PKG', script_output('rpm -qi kernel-default-extra', proceed_on_failure => 1));
 
+    record_info('KERNEL pkg', script_output('rpm -qa | grep kernel', proceed_on_failure => 1));
```
Comment (mdoucha) [inline lib/LTP/utils.pm:125]: `KERNEL DEFAULT PKG` above will show package info for `$kernel_pkg`. The info entry just needs to be renamed.

Comment (mdoucha) [reply lib/LTP/utils.pm:125]: Wait, never mind.

**Review by mdoucha — APPROVED** [review]
LGTM

File: lib/LTP/utils.pm (hunk 1 above)
Comment (pevik) [reply lib/LTP/utils.pm:125]: I'm sorry, I was just too lazy to put more code for proper label of `KERNEL DEFAULT PKG`. And `rpm -qa | grep kernel` might be useful for some new product.

**Pull request merged**